qa/vagrant/build.gradle

```diff
@@ -39,3 +54,18 @@ setupPackagingTest {
     expectedPlugins.setText(plugins.join('\n'), 'UTF-8')
   }
 }
+
+esvagrant {
+  testMainClass 'org.elasticsearch.packaging.PackagingMain'
```

Comment:
Could it be simpler to use a junit runner instead of having to define our own main class?

Reply:
We discussed this and I changed it to use junit as an entrypoint, as we likely would have ended up reimplementing a bunch of junit's runners' features